concepts/observability-custom-metrics/README.md

```diff
@@ -0,0 +1,93 @@
+However, it's an alpha project and is, in fact, abandoned. 
+
+### Pros
+
+* Easy to set up - just expose some Kyma Prometheus Operator features to end users
+
+### Cons
+
+* Since Prometheus Operator does not support annotation-based discovery of services, its CRD-based API (PodMonitor or ServiceMonitor) has to be exposed to end users. It is very flexible, but requires from end users advanced knowledge of Prometheus
+* Since the Prometheus config is under control of Kyma Prometheus Operator, it's not possible to use cardinality prevention alike Bomb Squad
+* Hard to guarantee Kyma monitoring stability since custom metrics will be pulled by the same Prometheus instance
+
+## Separate plain Prometheus
+
+Deploy a plain (non-operated) Prometheus server instance. Scraping custom workloads can be enabled by setting the following Pod annotations: 
+```yaml
+annotations:
+  prometheus.io/scheme: "https"
+  prometheus.io/scrape: "true"
+  prometheus.io/path: $(PROMETHEUS_METRICS_PATH)
+  prometheus.io/port: $(PROMETHEUS_METRICS_PORT)
+```
+
+Dividing Prometheus into Kyma Prometheus and Custom Worload Prometheus has a lot of advantages. However, we still have to let end users monitor Custom Workload Prometheus's stability and raise alerts if it hits the limits. We also have to suppress high cardinality metrics scraping (e.g. by using something similar to Bomb Squad)
+
+It makes sense to use a shared Grafana that queries both Prometheus instances. It is possible to achieve it without making any changes in the Grafana configuration, just by adding a custom datasource.
+
+### How to test it?
+
+```bash
+kubectl apply -f assets/simple-prometheus-setup.yaml # deploy Prometheus server and make it a Kyma Grafana datasource
+kubectl apply -f assets/workloads.yaml               # deploy a custom worload that exposes metrics
+kubectl apply -f assets/dashboard.yaml               # deploy Grafana dashboard
+```
+
+### Pros
+
+* Separating Kyma Prometheus, which is fully under our control, from Custom Worload Prometheus. In this setup, Kyma Prometheus remains unaffected by custom metrics hitting scraping limits, having high-cardinality, etc
+* Enabling metric scraping is can easily achieved by setting a set of annotations. In theory, it is possible to wtite a simple mutating webhook server to add those annotations automatically
+
+### Cons
+
+* Limited configuration flexibility (e.g. comparing to ServiceMonitors). According to the [official documentation](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#relabel_config), it's only possible to set scrape interval and scrape timeout on a per target basis
+* Second Prometheus instance needs additional cluster resources
+
+## Separate operated Prometheus (using Kyma Prometheus Operator)
+
+Use Kyma Prometheus Operator to deploy a second Prometheus instance. However, this approach combines disadvantages from both above-mentioned approaches: complex API and hard to prevent cardinality explosion.
```

Comment:
```suggestion
Use Kyma Prometheus Operator to deploy a second Prometheus instance. However, this approach combines disadvantages from both previously mentioned approaches: complex API and hard to prevent cardinality explosion.
```